source/connect.txt

```diff
@@ -1,34 +1,34 @@
-.. _node-connection:
+.. _node-connect:
```

Comment:
We try to keep old page labels intact, on the off-chance that a link in some other docs is still using it.

```suggestion
.. _node-connect:
.. _node-connection:
```

applies throughout